README.md

```diff
@@ -155,9 +166,9 @@ defmodule MyChat do
   end
 end
 
-{:ok, res} = MyChat.simple_chat("hi, how mutch is 1 + 2?")
+{:ok, res} = MyChat.simple_chat("hi, how much is 1 + 2?")
 
-IO.inspect res.main_response
+IO.inspect(res.main_response)
```

Comment:
Same. Would `res.main_response` return "3"? It would be handy to add the response to the readme.

Reply:
Same, it is some lines below

Reply:
Info is a bit below